src/main/java/org/kainos/ea/db/EmployeeDao.java

```diff
@@ -36,4 +36,32 @@ public List<Employee> getAllEmployees() throws SQLException {
 
         return employeeList;
     }
+
+    public DeliveryEmployee getDeliveryEmployeeById(int id) throws SQLException {
+        Connection c = DatabaseConnector.getConnection();
+        assert c != null;
+
+        Statement st = c.createStatement();
+        ResultSet rs = st.executeQuery("SELECT employee_id, first_name, " +
+                        "last_name, salary, bank_account_number, national_insurance_number " +
+                        "FROM delivery_employee " +
+                        "INNER JOIN employee " +
+                        "USING(employee_id) " +
+                        "WHERE employee_id ="+id);
```

Comment:
It's probably fine for this context, but could you see any issues with potentially appending the `id` directly to the end of the query?